src/VaccineTracker/TrackerForm.tsx

```diff
@@ -10,33 +10,33 @@ import {
 
 import { Results } from "../Results/Components/Results";
 import "../Styles/Form.css";
-import { Payload } from "../Interfaces/payload";
+import { requestPayload } from "../Interfaces/payload";
 
 import { getVaccineInfo } from "../VaccineTracker/store/vaccineTracker.actions";
 
 const mapStateToProps = (state: any) => {
   return {
     result: state.vaccineTracker.vaccineResults,
     isLoading: state.vaccineTracker.isLoading,
-    isResponseReceived: state.vaccineTracker.hasResponseReceived,
+    isResponseReceived: state.vaccineTracker.isResponseReceived,
   };
 };
+
 type Props = ReturnType<typeof mapStateToProps>;
 
 function Component(props: Props) {
   const dispatch = useDispatch();
   const formik = useFormik({
     initialValues: {
       pincode: "",
-      date: "",
     },
-    onSubmit: (values: Payload) => {
-      dispatch(getVaccineInfo(values));
+    onSubmit: (value: requestPayload) => {
+      dispatch(getVaccineInfo(value));
     },
   });
-  function Form() {
+  function trackVaccineForm() {
```

Comment:
If this is a function to return a JSX.element, I might make sense to make it a react component. There is not much difference but when things are transpiled, there might be some optimisation that can be done for components.